lib/manager/gomod/__snapshots__/artifacts.spec.ts.snap

```diff
@@ -31,7 +31,6 @@ Array [
         "GOFLAGS": "-modcacherw",
         "GONOPROXY": "noproxy.example.com/*",
         "GONOSUMDB": "1",
-        "GOPATH": "/tmp/renovate/cache/others/go",
```

Comment:
Any idea why this changes?

Reply:
This changed because previously we didn't mock `ensureCacheDir()`, but now we mock it. So it returns `undefined` and GOPATH is undefined.